src/Details.js

```diff
@@ -1,12 +1,48 @@
 import React from 'react';
-import {Link} from 'react-router-dom'
+import { Link } from 'react-router-dom';
 
-export default function Details () {
-    return(
-        <div>
-            <h1>Welcome to my details Page!!!!</h1>
-            <Link to='/'>Back to the Home Page</Link>
-        </div>
-    )
+export default class Details extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            message: 'Hello, this will be the details page for each Movie & TV Show! :)'
+        };
+    }
 
-}
\ No newline at end of file
+    componentDidMount() {
+        setTimeout(() => {
+            this.setState({ message: 'Coming soon! :)' });
+        }, 3000);
+    }
+    render() {
+        return (
+            <div>
+                <h1>{this.state.message}</h1>
+                <Link to='/'>Back to the Home Page</Link>
+            </div>
+        );
+
+    }
+}
+// export default function Details() {
```

Comment:
Please remove commented code!

If you ever check what you did in the past, you can review it using the commit history on gitHub :)

Reply:
I remove the commented code. Thanks